aggregator/src/proof_aggregation/config.rs

```diff
@@ -0,0 +1,130 @@
+use halo2_proofs::{
+    halo2curves::bn256::{Fq, Fr, G1Affine},
+    plonk::{Column, ConstraintSystem, Instance},
+};
+use snark_verifier::{
+    loader::halo2::halo2_ecc::{
+        ecc::{BaseFieldEccChip, EccChip},
+        fields::fp::FpConfig,
+        halo2_base::gates::{flex_gate::FlexGateConfig, range::RangeConfig},
+    },
+    util::arithmetic::modulus,
+};
+use zkevm_circuits::{
+    keccak_circuit::{KeccakCircuitConfig, KeccakCircuitConfigArgs},
+    table::KeccakTable,
+    util::{Challenges, SubCircuitConfig},
+};
+
+use crate::param::{ConfigParams, BITS, LIMBS};
+
+#[derive(Debug, Clone)]
+#[rustfmt::skip]
+/// Configurations for aggregation circuit.
+/// This config is hard coded for BN256 curve.
+pub struct AggregationConfig {
+    /// Non-native field chip configurations
+    pub base_field_config: FpConfig<Fr, Fq>,
+    /// Keccak circuit configurations
+    pub keccak_circuit_config: KeccakCircuitConfig<Fr>,
+    /// Instance for public input; stores
+    /// - accumulator from aggregation (12 elements)
+    /// - aggregated public inputs (136 elements):
+    ///     chain_id ||
```

Comment:
chain_id is now put at last for instance in accordance with https://github.com/scroll-tech/zkevm-circuits/blob/a6d627de6a3c203c4ac350d5eef1a6f85d6c4ce1/aggregator/src/proof_aggregation/circuit.rs#L360